docs/appendices/annotations.md

```diff
@@ -89,3 +89,35 @@ class \nodoc\Foo
    We don't want this class and its methods to appear in generated documentation
    """
 ```
+
+### `nosupertype`
+
+Recognised on objects(`actor`, `class`, `primitive`, `struct`). A type annotated with `nosupertype` will not be a subtype of any other type (except _), even if the type structurally provides an interface. If a `nosupertype` type has a provides list, a compiler error is reported. As a result, a `nosupertype` type is excluded from both nominal and structural subtyping.
```

Comment:
What's the meaning of `_` in this context?

Reply:
I lifted that directly from the RFC. I need to go look and see again what specifically `_` is in the type system.